src/Navigation.js

```diff
@@ -1,131 +1,49 @@
-                                    <Card.Header>
-                                        <Accordion.Toggle as={Link} to="/register" eventKey="0">Register</Accordion.Toggle>
-                                    </Card.Header>
-                                </Card>
-                            </Nav.Item>
-                            <Nav.Item>
-                                <Card>
-                                    <Card.Header>
-                                        <Accordion.Toggle as={Link} to="/login" eventKey="0" onClick={login}>Login</Accordion.Toggle>
-                                    </Card.Header>
-                                </Card>
-                            </Nav.Item>
-                        </>
-                        :
-                        <Nav.Item>
-                            <Card>
-                                <Card.Header>
-                                    <Accordion.Toggle as={Link} to="/" eventKey="0" onClick={logout}>Logout</Accordion.Toggle>
-                                </Card.Header>
-                            </Card>
-                        </Nav.Item>
-                    }
-                    {isAuth && <Nav.Item>
-                        <Card>
-                            <Card.Header>
-                                <Accordion.Toggle as={Link} to="/" eventKey="0">Welcome back, {user}!</Accordion.Toggle>
-                            </Card.Header>
-                        </Card>
-                    </Nav.Item>}
+      <div className="topnav">
+               <Navbar className="ml-auto" expand="lg" style={{borderTop:' 2px solid #54BCEB'}}>
+               <Navbar.Toggle aria-controls='responsive-navbar-nav' />
+               <Navbar.Collapse id="responsive-navbar-nav">
+               <Nav classname="mr-auto">
+                 <div className="nav-items">
+                      <Link to="\" style={{textDecoration:"none"}}>
+                         <li >About Us</li>
+                      </Link>
+                      <Link to="\" style={{textDecoration:"none"}}>
+                          <li >Services</li>
+                      </Link>
+                      <Link to="\" style={{textDecoration:"none"}}>
+                          <li >Membership</li>
+                      </Link>
+                      <Link to="\" style={{textDecoration:"none"}}>
+                          <li>FAQs</li>
+                      </Link>
+                      <Link to="\" style={{textDecoration:"none"}}>
+                          <li>Blogs</li>
+                      </Link>
+                      <Link to="\" style={{textDecoration:"none"}}>
+                          <li>Contact Us</li>
+                      </Link>
+                      <Link to="\" style={{textDecorationColor: "blue",textDecoration:"none"}}>
+                          <li >Sign In</li>
+                      </Link>
+                      <Link to="\" style={{textDecoration:"none"}}>
```

Comment:
@stableapple You can use a class instead of adding the inline styles to every link.